plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/commands/SQLCommandInclude.java

```diff
@@ -77,76 +72,113 @@ public SQLControlResult handleCommand(@NotNull DBRProgressMonitor monitor, @NotN
         }
 
         // Check for nested inclusion
-        for (SQLScriptContext sc = scriptContext; sc != null ;sc = sc.getParentContext()) {
+        for (SQLScriptContext sc = scriptContext; sc != null; sc = sc.getParentContext()) {
             if (sc.getSourceFile() != null && sc.getSourceFile().equals(incFile)) {
                 throw new DBException("File '" + fileName + "' recursive inclusion");
             }
         }
 
-        final String fileContents;
-        try (InputStream is = Files.newInputStream(incFile)) {
-            Reader reader = new InputStreamReader(is, getResourceEncoding());
-            fileContents = IOUtils.readToString(reader);
-        } catch (IOException e) {
-            throw new DBException("IO error reading file '" + fileName + "'", e);
+        return getSqlControlResult(
+            scriptContext,
+            incFile
+        );
+    }
+
+    @NotNull
+    private SQLControlResult getSqlControlResult(
+        @NotNull SQLScriptContext scriptContext,
+        @NotNull Path finalIncFile
```

Comment:
Likely a leftover from the old code, plus naming

```suggestion
        @NotNull Path includeFile
```